<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="welcomeLabel">
     <property name="styleSheet">
      <string>font-size: 20px; font-weight: bold; color: #333;</string>
     </property>
     <property name="text">
      <string>Welcome back!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="styleSheet">
      <string>padding: 5px; font-size: 14px;</string>
     </property>
     <property name="placeholderText">
      <string>Enter your username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="styleSheet">
      <string>padding: 5px; font-size: 14px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Enter your password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="rememberMeCheckBox">
     <property name="text">
      <string>Remember me</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="styleSheet">
      <string>background-color: #58b368; color: white; font-weight: bold; padding: 10px; font-size: 16px; border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="styleSheet">
      <string>font-size: 14px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
